tests/js/server/aql/aql-index-hints.js

```diff
@@ -451,7 +468,27 @@ function indexHintSuite() {
             assertNotEqual(indexHint[1].indexOf(node.indexes[0].name), -1);
         });
       });
-    }
+    },
+
+    testForceAnotherIndexWithGeoPresent: function () {
+      const query = `FOR d IN ${cn2}  OPTIONS {indexHint: "${invertedIdxName}", forceIndexHint: true, waitForSync: true} 
```

Comment:
waitForSync seems to be useless here, as no data is being read from the index.
```suggestion
      const query = `FOR d IN ${cn2}  OPTIONS {indexHint: "${invertedIdxName}", forceIndexHint: true } 
```

Reply:
Now we need this as we actually run query